Ds3/Runtime/Network.cs

```diff
@@ -196,14 +196,29 @@ private HttpWebRequest CreateRequest(Ds3Request request, Stream content)
                 httpRequest.ContentLength = request.GetContentLength();
                 if (content != Stream.Null)
                 {
-                    using (var requestStream = httpRequest.GetRequestStream())
+                    var requestStream = httpRequest.GetRequestStream();
```

Comment:
This should stay in a using block in the event that the `content.Seek` on line `202` throws an exception or the `Flush` on line `221`